bundles/org.openhab.binding.wemo/src/main/java/org/openhab/binding/wemo/internal/handler/AbstractWemoHandler.java

```diff
@@ -12,22 +12,23 @@
  */
 package org.openhab.binding.wemo.internal.handler;
 
+import org.eclipse.jdt.annotation.NonNullByDefault;
+import org.eclipse.jdt.annotation.Nullable;
 import org.openhab.binding.wemo.internal.http.WemoHttpCall;
 import org.openhab.core.thing.Thing;
 import org.openhab.core.thing.binding.BaseThingHandler;
 
 /**
- *
- * @author Stefan Triller
- *
+ * @author Stefan Triller - Initial contribution
  */
+@NonNullByDefault
 public abstract class AbstractWemoHandler extends BaseThingHandler {
 
     public AbstractWemoHandler(Thing thing) {
         super(thing);
     }
 
-    protected WemoHttpCall wemoHttpCaller;
+    protected @Nullable WemoHttpCall wemoHttpCaller;
```

Comment:
Why not have this get passed in through the constructor so that you don't have to make it nullable?

Reply:
Sorry, don't understand you, can you give an example please.

Reply:
```
public abstract class AbstractWemoHandler extends BaseThingHandler {

    protected WemoHttpCall wemoHttpCaller;

    public AbstractWemoHandler(Thing thing, WemoHttpCall wemoHttpCaller) {
        super(thing);
        this.wemoHttpCaller = wemoHttpCaller;
    }

```

Reply:
Done